satellite-placement/README.rst

```diff
@@ -0,0 +1,43 @@
+Satellite Placement Demo
+========================
+A demo on how to group a fleet of satellites into constellations of size x,
+such that the average coverage of each constellation is maximized.
```

Comment:
Yes, I too had hesitated with "fleet". I didn't want to use the word "constellation" multiple times in case the reader mixed up the the context.

Anyway, I shall fix this.